tests/90jira/SYN-482.pl

```diff
@@ -0,0 +1,52 @@
+multi_test "Limit on room/initialSync is reached over federation (SYN-482)",
+    requires => [qw( make_test_room local_users remote_users)],
```

Comment:
This looks like a merge of some very old code; because it's using `local_users` and `remote_users`. That hasn't been around for a month or so at least.

Take a look around instead at some of the other tests, how they use `local_user_fixture()` and `remote_user_fixture()` - you probably want to use those instead.